src/Components/Error/Error.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import './Error.css'
 
-const Error = errDets => (
+const Error = ({status, statusText}) => (
   <div className='error'>
     <h1>Sorry!</h1>
-    {errDets.status && <h2>{errDets.status}</h2>}
-    {errDets.status.toString()[0] === 4 && <h2>Something went wrong!</h2>}
-    {errDets.status.toString()[0] === 5 && <h2>Something went wrong with the server!</h2>}
+    {status && <h2>{status}</h2>}
+    {status.toString()[0] === '4' && <h2>{statusText}</h2>}
+    {status.toString()[0] === '5' && <h2>{statusText}</h2>}
```

Comment:
Right now I'm just using conditional rendering to generate JSX to display. How does this look?

Reply:
I think this looks perfect. If an issue shows up should be super fixable.